documentation/public/user-guide/rust-crate/how-to-parse-a-file.md

```diff
@@ -0,0 +1 @@
+--8<-- "crates/solidity/inputs/language/snippets/under-construction.md"
```

Comment:
nit: I suggest putting each page into its own folder. For example, `how-to-parse-a-file/index.md` instead of `how-to-parse-a-file.md`, as it makes it easier to understand hierarchy when browsing the file tree. It is also common for pages to evolve over time to have their own resources next to it (graphs/images/snippets/etc...) in the dedicated folder.